### PR TITLE
Import owner private keys script, updated generate key pair script

### PR DESCRIPTION
**Pull request by lorenzoking**

### CODE REVIEW

File: tools/import-owner-private-keys.sh [hunk 1]
```diff
@@ -1,6 +1,100 @@
...
+}
+
+
+#============================MAIN CODE=================================
+
+ensureWeAreUser
+untarKeyFiles
+genKeyStore
+insertKeys
+
```
Comment (bmpotter) [inline tools/import-owner-private-keys.sh:98]: It appears that this script gives no output in the successful case. You are correct that this is the linux convention for short commands that are run interactively (no news is good news). But since this script is run in the background inside the `sdo-owner-services` container, and usually only run once, and is critical to the rest of the operation of the container, it would be nice to see a msg that it was success in the `sdo-owner-services` log.

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: tools/import-owner-private-keys.sh [hunk 2]
```diff
@@ -1,6 +1,100 @@
...
+if [[ "$1" == "-h" || "$1" == "--help" ]]; then
+    cat << EndOfMessage
+Usage: ${0##*/} [<owner-keys-tar-file>]
+
+Arguments:
+  <owner-keys-tar-file>  A tar file containing the 3 private keys and associated 3 certs
+
+Required environment variables:
+  SDO_KEY_PWD - The password for your generated keystore. This password must be passed into run-sdo-owner-services.sh if you are passing in your own master keystore. This will allow it to be mounted to $containerHome/ocs/config/application.properties/fs.owner.keystore-password
```
[the remark this answers is not in the document]
Comment (lorenzoking) [reply tools/import-owner-private-keys.sh:12]: SDO_KEY_PWD is passed in as an environment variable for this script. In `run-sdo-owner-services.sh` if you pass in your own master keystore then you must also pass in the corresponding SDO_KEY_PWD env variable. Once that is passed in it will update ocs/config/application.properties/fs.owner.keystore-password with whatever password you decided to use. This is the same password that will be used by this script so that it can insert the keypairs inside the master keystore.

In short you should be able to use our master keystore or even pass in your own with a custom password and everything should continue to work

**Review by bmpotter — APPROVED** [review]
